react/booksManager.js

```diff
@@ -0,0 +1,70 @@
+import {Component} from 'react'
+
+export class BooksManager extends Component {
+  constructor(props) {
+    super(props)
+    this.clearInput()
+  }
+
+  createBook() {
+    const options = {
+      variables: {
+        name: this.state.name,
+        authors: this.state.authors.split(',')
+      },
+      refetchQueries: [{query: this.props.onGet}]
+    }
+
+    this.props.onCreate(options).then(this.clearInput.bind(this))
+  }
+
+  deleteBook() {
+    const options = {
+      variables: {
+        id: this.state.id
+      },
+      refetchQueries: [{query: this.props.onGet}]
+    }
+
+    this.props.onDelete(options).then(this.clearInput.bind(this))
+  }
+
+  clearInput() {
+    this.state = Object.assign({}, {
+      name: '',
+      authors: '',
+      id: ''
+    })
+  }
+
+  render() {
+    const {title, books} = this.props
+
+    return (
+      <div>
+        <h2>{title}</h2>
+        <h3>Books in the Library</h3>
+        {books && books.map(({id, name, authors}) => {
+          return <div key={id}>
+                    <b>{name || 'Name me, please'}</b>
```

Comment:
Just a few more minor things I've just noticed:

1. The indentation here is off. This line should only be a single level higher than the previous one, relative to `return`, not to `<div`. The other lines should be updated in the same way, ending like this:

```js
        {books && books.map(({id, name, authors}) => {
          return <div key={id}>
              <b>{name || 'Name me, please'}</b>
              <br/>Authors(split by ','): {(authors && authors.map((authors) => `${authors}, `)) || 'Give me a father, please'}
              <br/>ID (click to select):<div onClick={event => this.setState({id: id})}>{id || 'I should have an ID, this is embarrassing'}</div>
              <br/>
          </div>
        })}
```
2. We use the name of a file exporting a single component as the name of the component itself. So this should be `BooksManager.js`, not `booksManager.js`.